src/config/tests.rs

```diff
@@ -56,6 +56,7 @@ fn setup_config_test() -> Configuration {
             filter_word_count = [994, 992]
             filter_line_count = [34]
             filter_status = [201]
+            update_app = false
```

Comment:
since we're not going to put update_app into the config file, we can remove all of the changes in this file